frontend/src/app/modules/settings/team/edit-team/edit-team.component.ts

```diff
@@ -51,7 +51,15 @@ export class EditTeamComponent extends BaseComponent implements OnDestroy {
                 .pipe(this.untilThis)
                 .subscribe({
                     next: (team) => {
-                        this.team = team;
```

Comment:
Why separate fields intialization is needed?

Reply:
because the TeamPreferencesComponent did not receive the current Team